src/App.js

```diff
@@ -1,19 +1,45 @@
 import Clothing from './components/Clothing'
 import styled from 'styled-components/macro'
-import data from './mockup-data'
+import mockupData from './mockup-data'
 
 function App() {
+  let data
+
+  if (localStorage.getItem('localClothing')) {
+    data = JSON.parse(localStorage.getItem('localClothing'))
+  } else {
+    localStorage.setItem('localClothing', JSON.stringify(mockupData))
+    data = JSON.parse(localStorage.getItem('localClothing'))
+  }
+
+  const handleBookmark = (bookmarkActive) => {
+    const index = data.findIndex((card) => card.id === bookmarkActive.id)
+
+    localStorage.setItem(
+      'localClothing',
+      JSON.stringify([
+        ...data.slice(0, index),
+        bookmarkActive,
+        ...data.slice(index + 1),
+      ]),
+    )
+  }
```

Comment:
In your function `handleBookmark` you are currently only updating the localStorage and not the `data`. I would suggest making `data` a state and then updating the state here as well. How about naming the state `[clothes, setClothes]`?
```suggestion
  const handleBookmark = (bookmarkActive) => {
    const index = data.findIndex((card) => card.id === bookmarkActive.id)
    const newClothingArray = [
        ...clothes.slice(0, index),
        bookmarkActive,
        ...clothes.slice(index + 1),
      ]
      
    setClothes(newClothingArray)
    localStorage.setItem(
      'localClothing',
      JSON.stringify(newClothingArray),
    )
  }
```